lib/controllers/main_info.dart

```diff
@@ -51,6 +51,13 @@ class MainInfo {
 
   String get outputPath => configuration.outputPath;
 
+  Map<String, dynamic> amplitudMap = {
```

Comment:
Small typos throughout the PR, it should be `amplitude` 👍🏾

Reply:
Oh snap, thanks!